src/AzureRMTools.ts

```diff
@@ -492,6 +519,31 @@ export class AzureRMTools {
         startArmLanguageServer();
     }
 
+    private async updateParamFileInStatusBar(): Promise<void> {
+        const activeDocument = vscode.window.activeTextEditor?.document;
+        if (activeDocument) {
+            const deploymentTemplate = this.getDeploymentTemplate(activeDocument);
+            if (deploymentTemplate) {
+                const paramFileUri = findMappedParamFileForTemplate(activeDocument.uri);
+                if (paramFileUri) {
+                    const doesParamFileExist = await fse.pathExists(paramFileUri?.fsPath);
+                    let text = `Parameters: ${getFriendlyPathToParamFile(activeDocument.uri, paramFileUri)}`;
+                    if (!doesParamFileExist) {
+                        text += " $(error) Not found";
```

Comment:
>(error) [](start = 35, length = 7)

Where does this come from?

Reply:
https://microsoft.github.io/vscode-codicons/dist/codicon.html

This is documented by vscode.